<launch>

  <arg name="input_detected_boxes" default="/tlr_ssd_detector/output_stamped_roi" />
  <arg name="input_image" default="/image_raw" />
  <arg name="input_stamped_roi" default="/feat_proj_bbox/output_projected_roi" />

  <node name="tlr_kcf_tracker"
        pkg="nodelet" type="nodelet"
        output="screen"
        args="standalone trafficlight_recognizer/KcfTrackerNode">
    <remap from="~input_detected_boxes" to="$(arg input_detected_boxes)" />
    <remap from="~input_image" to="$(arg input_image)" />
    <remap from="~input_stamped_roi" to="$(arg input_stamped_roi)" />
    <rosparam subst_value="true">
      kernel_sigma           : 0.5
      cell_size              : 4
      num_scales             : 7
      interpolation_frequency: 1
      buffer_size            : 100
      approximate_sync       : true
    </rosparam>
  </node>

</launch>
